.github/workflows/check-labels-new.yml

```diff
@@ -15,12 +15,19 @@ jobs:
           ref: ${{ github.event.pull_request.head.ref }}
           repository: ${{ github.event.pull_request.head.repo.full_name }}
 
-      - name: Pull image
-        env:
-          IMAGE: chevdor/ruled-labels:latest
-        run: |
-          docker pull $IMAGE
-          docker run --rm $IMAGE --version
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          override: true
+          components: ruled-labels
```

Comment:
I think this option will be much longer (we need to build and install ruled-labels) everytime, compared to using a (pre-built) docker image. Did you check the timing ?

Reply:
It may however still be fast is the cache works good. It would be good to check though since checking the labels ideally should be very fast.